light_curves/code_src/plot_functions.py

```diff
@@ -96,6 +96,66 @@ def create_figures(df_lc, show_nbr_figures, save_output):
             
     print("Done")
     return(True)
+    
+def create_figure(df_lc, index, save_output):
+    '''
+    Creates a figure of the lightcurve for a specific object in df_lc, identified by index.
+    
+    Parameters
+    ----------
+    df_lc : DataFrame
+        DataFrame containing lightcurve objects from which to create the lightcurve figure.
+        
+    index : int
+        Index of the object to plot. This is not the object ID but the position in the list of grouped objects.
```

Comment:
I had forgotten that objectid is just an index, I was thinking it comes from the main catalog where the RA/DEC were read and thought it would be difficult to figure out which ids exist.